fireplace-swing/src/main/java/io/github/bric3/fireplace/flamegraph/FlameGraph.java

```diff
@@ -325,14 +364,15 @@ public void mouseMoved(MouseEvent e) {
             }
 
             canvas.getFlameGraphPainter()
-                  .ifPresent(fgp -> fgp.hoverFrameAt(
-                          (Graphics2D) view.getGraphics(),
-                          point,
-                          frame -> {
-                              canvas.setToolTipText(frame);
-                              scrollPane.repaint();
-                          }
-                  ));
+                    .ifPresent(fgp -> fgp.hoverFrameAt(
+                            (Graphics2D) view.getGraphics(),
+                            canvas.getBounds(),
+                            point,
+                            frame -> {
+                                canvas.setToolTipText(frame);
+                                scrollPane.repaint();
+                            }
+                    ));
```

Comment:
For reference might conflict with #34 https://github.com/bric3/fireplace/pull/34/files#diff-e367861376ff2c84840ac2b45bbf62c25e02fc13bbe3f5a44f6d5372bed5b3dbR329-R335

Let's merge #34 after this one as it's much simpler.